tests/dialects/test_spark.py

```diff
@@ -234,7 +234,7 @@ def test_spark(self):
         self.validate_identity("first_value(col, true)", "FIRST_VALUE(col) IGNORE NULLS")
         self.validate_identity("last(col, true)", "LAST(col) IGNORE NULLS")
         self.validate_identity("last_value(col, true)", "LAST_VALUE(col) IGNORE NULLS")
-        self.validate_identity("DESCRIBE EXTENDED db.table")
+        self.validate_identity("DESCRIBE EXTENDED db.tbl")
```

Comment:
how come these tests changed?

Reply:
tried adding them to spark but then a bunch of this stuff changed